test/e2e/framework/pod/wait.go

```diff
@@ -302,7 +301,7 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 				return fmt.Errorf("Unexpected error: %v", err)
 			}
 			if !done {
-				conditionNotMatch = append(conditionNotMatch, format.Pod(&pod))
+				conditionNotMatch = append(conditionNotMatch, fmt.Sprintf("%s_%s(%s)", pod.Name, pod.Namespace, pod.UID))
```

Comment:
The original of here is like
```
 30 // Pod returns a string representing a pod in a consistent human readable format,
 31 // with pod UID as part of the string.
 32 func Pod(pod *v1.Pod) string {
 33         return PodDesc(pod.Name, pod.Namespace, pod.UID)
 34 }
 35
 36 // PodDesc returns a string representing a pod in a consistent human readable format,
 37 // with pod UID as part of the string.
 38 func PodDesc(podName, podNamespace string, podUID types.UID) string {
 39         // Use underscore as the delimiter because it is not allowed in pod name
 40         // (DNS subdomain format), while allowed in the container name format.
 41         return fmt.Sprintf("%s_%s(%s)", podName, podNamespace, podUID)
 42 }
```
/lgtm